addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1546,7 +1555,7 @@ msgid "Screensaver"
 msgstr ""
 
 msgctxt "#361"
-msgid "R. Slideshow"
+msgid "R. slideshow"
```

Comment:
I never used picture section in Kodi - does anybody know what that "R." actually stands for?